src/import-data/ImportData.ts

```diff
@@ -107,10 +107,20 @@ export const ImportData = (props: ImportDataProps): ReactNode => {
     if (workspace.cloudPlatform === 'Azure') {
       // find wds for this workspace
       const wdsUrl = await Ajax().Apps.listAppsV2(workspace.workspaceId).then(resolveWdsUrl);
-      const wdsDataTableProvider = new WdsDataTableProvider(workspace.workspaceId, wdsUrl);
+      const { namespace, name } = workspace;
 
       // call import snapshot
-      await wdsDataTableProvider.importTdr(workspace.workspaceId, importRequest.manifestUrl);
+      const { jobId } = await Ajax().WorkspaceData.startImportJob(wdsUrl, workspace.workspaceId, {
+        url: importRequest.manifestUrl.toString(),
+        type: 'TDRMANIFEST',
+      });
+      const newJob: AzureAsyncImportJob = {
+        targetWorkspace: { namespace, name },
+        jobId,
+        wdsProxyUrl: wdsUrl,
+      };
+      asyncImportJobStore.update((previousJobs) => [...previousJobs, newJob]);
```

Comment:
There's a helper method for this.
```suggestion
      asyncImportJobStore.update(Utils.append(newJob));
```

Reply:
Thanks!